counsel.el

```diff
@@ -3127,6 +3127,26 @@ otherwise continue prompting for tags."
 ;;;###autoload
 (defalias 'counsel-org-goto #'counsel-outline)
 
+(defcustom counsel-org-goto-all-outline-path-prefix nil
+  "When the value is 'file, include the file name (without
+directory) into the outline path candidate.
+
+When 'full-file-path, include the full file name.
+
+When 'buffer-name, include the buffer name."
```

Comment:
In Elisp docstrings, the first line should be a summary of the definition's purpose, and symbols should be quoted with backtick and apostrophe. See other definitions in this project for examples, and [`(elisp) Documentation Tips`](https://www.gnu.org/software/emacs/manual/html_node/elisp/Documentation-Tips.html) for a more complete explanation of Elisp conventions.